habitat-lab/habitat/sims/habitat_simulator/sim_utilities.py

```diff
@@ -824,6 +964,58 @@ def ontop(
     return ontop_object_ids
 
 
+def on_floor(
```

Comment:
so to be clear, this is more implementing on_navmesh right?
Maybe we can add a comment that the floor isn't a unique receptacle, but approximated as the whole navmesh right now?
Sorry if I'm misunderstanding the code.

Reply:
Yes, this is navmesh based solution. Happy to buff up the documentation a bit here.